src/Microsoft.DotNet.Interactive/Server/StandardIOKernelServer.cs

```diff
@@ -39,7 +39,8 @@ public class StandardIOKernelServer : IDisposable
                 {
                     await DeserializeAndSendCommand(line);
                 }),
-                _kernel.KernelEvents.Subscribe(WriteEventToOutput)
+                _kernel.KernelEvents.Subscribe(WriteEventToOutput),
```

Comment:
Should we be tracking the subscription for disposal?

Reply:
we do, that code  is inside the CompositeDisposabe contructor

Reply:
Subscribe returns a disposable and we're not keeping a reference to it.